src/ngx-gallery-action.component.ts

```diff
@@ -1,30 +1,35 @@
-import { ChangeDetectionStrategy, Component, Input, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
-import { NgxGalleryPreviewComponent } from "./ngx-gallery-preview.component";
-import { NgxGalleryScrollOverviewComponent } from "./ngx-gallery-scroll-overview/ngx-gallery-scroll-overview.component";
+import { ChangeDetectionStrategy, Component, Input, Output, EventEmitter, ViewChild } from '@angular/core';
+import { NgxGalleryPreviewComponent } from './';
+import { ZoomPosition } from './ngx-gallery-scroll-overview/zoomPosition';
 
 @Component({
     selector: 'ngx-gallery-action',
     template: `
-        <i class="ngx-gallery-icon {{ icon }}" [class.ngx-gallery-icon-disabled]="disabled"
+        <div class="ngx-gallery-icon {{ icon }}" [class.ngx-gallery-icon-disabled]="disabled"
             aria-hidden="true"
             title="{{ titleText }}"
             (click)="handleClick($event)" >
-           <ngx-gallery-scroll-overview #scrollOverviewComponent [gallery]="gallery" *ngIf="rectangleScroll"> </ngx-gallery-scroll-overview>
-        </i>`,
+           <ngx-gallery-scroll-overview #scrollOverviewComponent (onZoomChanged)="zoomChanged()" [zoomPosition] = "zoomPosition" *ngIf="showScrollOverview"> </ngx-gallery-scroll-overview>
+        </div>`,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class NgxGalleryActionComponent {
-
     @Input() icon: string;
     @Input() disabled = false;
     @Input() titleText = '';
-    @Input() rectangleScroll: boolean;
+    @Input() showScrollOverview: boolean;
+    @Input() zoomPosition: ZoomPosition;
     @Input() gallery: NgxGalleryPreviewComponent;
 
     @Output() onClick: EventEmitter<Event> = new EventEmitter();
+    @Output() onZoomChanged = new EventEmitter();
```

Comment:
Change name to `zoomChanged` (I know, they use `onClick` for their event emitter but this just makes no sense.